Solutions/SevcoSecurity/Data/Solution_SevcoSecurity.json

```diff
@@ -0,0 +1,15 @@
+  {
+    "Name": "SevcoSecurity",
+    "Author": "Sevco Security - support@sevco.io",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/sevcosecurity.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "The Sevco Platform - Devices connector allows you to easily connect your Sevco Device Assets with Microsoft Sentinel, to view dashboards, create custom alerts, and improve investigation. This gives you more insight into your organization’s assets and improves your security operation capabilities.\n\n[For more information >​](https://docs.sev.co/docs/microsoft-sentinel-inventory)",
+    "Data Connectors": [
+      "Data Connectors/Connector_SevcoSecurity.json"
+    ],
+    "BasePath": "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SevcoSecurity/",
```

Comment:
If I change the path, then the packaging tool cannot find the input files.

I asked a question regarding this on May 2 and did not get a response.

Reply:
The error is:
```
Downloading C:\Github\Azure-Sentinel\Solutions\SevcoSecurity/Data Connectors/Connector_SevcoSecurity.json
Failed to download C:\Github\Azure-Sentinel\Solutions\SevcoSecurity/Data Connectors/Connector_SevcoSecurity.json -- Please ensure that it exists in C:%5CGithub%5CAzure-Sentinel%5CSolutions%5CSevcoSecurity/
Downloading /Users/miah/Documents/dev/Azure-Sentinel/Tools/Create-Azure-Sentinel-Solution/V2/../../../Solutions/SevcoSecurity/SolutionMetadata.json
Failed to write output file /Users/miah/Documents/dev/Azure-Sentinel/Solutions/SevcoSecurity/Package/createUiDefinition.json
```

Reply:
Hello @MiahClark can we setup a call for this, it would be helpful for the better resolution of the Solution.
My email id is v-prasadboke, Can you please tell your time zone and your availability according to your time so that we can setup a call for the issues you are facing